recipes/default.rb

```diff
@@ -176,7 +176,7 @@
 
 ###
 # Get some banners up and running!
-# See https://github.com/18F/ubuntu/blob/master/hardening.md#ssh-settings
+# See https://github.com/18F/ubuntu-lts/blob/master/hardening.md#ssh-settings
 ###
```

Comment:
Should this point to https://github.com/18F/ubuntu-lts/blob/master/hardening.md#banners?